packages/mgt-components/src/components/mgt-todo/mgt-todo.ts

```diff
@@ -126,14 +126,26 @@ export class MgtTodo extends MgtTasksBase {
     if (tasks && this.taskFilter) {
       tasks = tasks.filter(task => this.taskFilter(task));
     }
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+    const completedTasks = tasks.filter(task => (TaskStatus as any)[task.status] === TaskStatus.completed);
```

Comment:
Needing to do this feels wrong.
Could we make TaskStatus a string union type instead and avoid all this casting e.g.
```
export type TaskStatus = 'notStarted' | 'inProgress' | 'completed' | 'deferred' | 'waitingOnOthers';
```